From: Evgeni Golov <evgeni@debian.org>
Date: 2011-04-26
Bug-Debian: http://bugs.debian.org/525415
Subject: Don't include random kernel headers in CFLAGS
 These are not needed for building and may have random results

Index: v86d-0.1.10/Makefile
===================================================================
--- v86d-0.1.10.orig/Makefile	2011-04-26 09:35:16.724564809 +0200
+++ v86d-0.1.10/Makefile	2011-04-26 09:35:46.804572140 +0200
@@ -3,14 +3,12 @@
 .PHONY: clean install install_testvbe
 
 INSTALL = install
-KDIR   ?= /lib/modules/$(shell uname -r)/source
 
 ifeq ($(call config_opt,CONFIG_KLIBC),true)
 	export CC = klcc
 endif
 
 CFLAGS ?= -Wall -g -O2
-CFLAGS += -I$(KDIR)/include
 
 LDLIBS += -lx86
 V86OBJS = v86_lrmi.o v86_common.o
